services/order.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
```

Comment:
The `date` parameter should be a `datetime` object instead of a string to ensure proper handling of date and time values.